reviewing/plugin-commands-licenses/src/outputRenderer.ts

```diff
@@ -112,13 +112,46 @@ function renderLicensesTable (
   for (let i = 0; i < columnNames.length; i++)
     columnNames[i] = chalk.blueBright(columnNames[i])
 
+  const data = [
+    columnNames,
+    ...deduplicateLicensesPackages(sortLicensesPackages(licensePackages))
+      .map((licensePkg) => columnFns.map((fn) => fn(licensePkg))),
+  ]
+  let detailsColumnMaxWidth = 40
+  let packageColumnMaxWidth = 0
+  let licenseColumnMaxWidth = 0
+  if (opts.long) {
+    detailsColumnMaxWidth = data.slice(1).reduce((maxWidth, row) => {
+      const texts = row[2].split('\n')
+      const link = texts.find((line) => line.startsWith('https'))
+      // Use the package link to determine the width of the details column
```

Comment:
why not restructuring the code so that you get the link length before it gets formatted to the details text? You have this data in an object. So, why not getting it from there instead?

Reply:
Thanks for your reminder.